source/user-guide/map/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _map-viewer-as-user:
+
+Maps and dataset visualisation
+==============================
+
+The map component in GeoNetwork serves two goals:
+
+- As part of the evaluation of relevance users can visualise a resource discovered in the catalogue in an interactive map viewer to preview the contents.
+
+- GeoNetwork can be used as a mapping application to create maps that can be printed or shared with others. The maps itself can even be registered in the catalogue as assets.
+
+.. Note::
+
+    In stead of using the embedded map viewer, GeoNetwork offers functionality to use an externa map viewer (for example an existing WebGIS framework in your organisation). An administrator can configure GeoNetwork in such a way that it is connected to an external web map application. In such a configuration all map interaction is delegated to the external application.
+
+Map viewer
+----------
+
+Map viewer can be opened from a metadata detais view by the 'add to map' button or directy from the map tab in the main menu.
```

Comment:
The map viewer can be opened from a metadata details view using the 'add to map' button or directly from the map tab (several typos)